examples/customMSWServer.test.js

```diff
@@ -0,0 +1,64 @@
+// @ts-check
+/**
+ * @typedef
+ */
+import createJWKSMock from 'mock-jwks'
+import { createApp } from './api.js'
+import supertest from 'supertest'
+import { beforeAll, beforeEach, describe, expect, test } from 'vitest'
+import { setupServer } from 'msw/node'
+
+const jwksMock = createJWKSMock('https://levino.eu.auth0.com')
+const app = createApp({
+  jwksUri: 'https://levino.eu.auth0.com/.well-known/jwks.json',
+})
+
+describe('Some tests for authentication for our api', () => {
+  /** @type {import('msw/node').SetupServerApi} */
+  let mswServer
+  beforeAll(() => {
+    mswServer = setupServer()
+    mswServer.listen({
+      onUnhandledRequest: 'bypass', // We silence the warnings of msw for unhandled requests. Not necessary for things to work.
+    })
+    return () => mswServer.close()
+  })
```

Comment:
This is a bit to simulate the usual usage [recommended](https://mswjs.io/docs/integrations/node#test-runner) by `msw`.